src/main/java/io/lakefs/routerfs/PathMapper.java

```diff
@@ -279,11 +257,12 @@ public String toString() {
      * A class that represents a parsed Path mapping configurations supported by RouterFileSystem.
      * Mapping configuration form is: routerfs.mapping.${fromFsScheme}.${mappingIdx}.${replace/with}=prefix.
      */
+    @Getter
```

Comment:
A bit less scary here because it's a POJO, but still unpleasantly scary.

Reply:
Why are we using the @Getter annotation in different ways? that is, in Mapping config the annotation is at the class level as opposed to PathMapping where we use it at the data member level?

Reply:
You can see the discussion about this [here](https://github.com/treeverse/hadoop-router-fs/pull/7/commits/b6147be0ff46efaf1b764749e4852c72bfb47e79)...
Basically, @arielshaqed raised a possible leak of information from this class, so I changed to more specific getter annotations, while `MappingConfig` is a POJO thus less of a risk...